ClemBot.Bot/bot/cogs/tags_cog.py

```diff
@@ -285,6 +288,74 @@ async def transfer(self, ctx, name: str, user: discord.User):
         embed.set_footer(text=self.get_full_name(author), icon_url=author.display_avatar.url)
         await ctx.send(embed=embed)
 
+# Tag prefix functions
+
+    
+    #@ext.group(case_insensitive=True, invoke_without_command=True, aliases=['tagprefixs'])
+    @tag.group(invoke_without_command=True, case_insensitive=True)
+    @ext.long_help(
+        'Lists the current tag prefix or configures the command prefix that the bot will respond too'
+    )
+    @ext.short_help('Configure a custom command tag prefix')
+    @ext.example(('tag prefix', 'tag prefix ?', 'tag prefix >>'))
+    async def prefix(self, ctx, *, tagprefix: t.Optional[str] = None):
+        # get_prefix returns two mentions as the first possible prefixes in the tuple,
+        # those are global so we dont care about them
+        tag_prefixes = (await self.bot.get_tag_prefix(ctx.message))
+
+        if not tagprefix:
+            embed = discord.Embed(title='Current Active Tag Prefixes',
+                                  description=f'```{", ".join(tag_prefixes)}```',
+                                  color=Colors.ClemsonOrange)
+
+            return await ctx.send(embed=embed)
+
+        if tagprefix in await self.bot.get_tag_prefix(ctx.message):
```

Comment:
we should reuse tag_prefixes here, this is making another api call which is super expensive